scripts/jtsync/jtsync.rb

```diff
@@ -128,11 +135,11 @@ def parse_job_from_cli
   opts.order!
   raise "Either job or matrix is required" if job.type.nil?
 
-  Job.for(job.service, job.type, job.name, job.project, ARGV.pop)
+  [board_id, Job.for(job.service, job.type, job.name, job.project, ARGV.pop)]
 end
 
-def board
-  @trello_board ||= Trello::Board.find(TRELLO_BOARD)
+def board(board_id)
+  @trello_board ||= Trello::Board.find(board_id)
```

Comment:
Another coding style point:

Here `@trello_board` is *effectively* a global variable, which is usually a code smell.  In a small script it's probably not a big issue, but it's still worth being aware of the fact that avoiding passing the value between methods via arguments hides the scope of this variable (i.e. which methods use it).

Also, if you are going to store state globally, then there is no need to require the caller to provide `board_id` each time or use the `||=` trick - instead you could simply set `@trello_board` the first time you obtain `board_id`, and then do

    def board
      @trello_board
    end

This removes the need to keep passing `board_id` around between methods - the current usage defeats the original point of using a global, which is to reduce the number of things being passed around.